### DISCUSSION TITLE
Regarding having more fields in the Weblate Translate interface

### DISCUSSION THREAD

**Discussion by prabodhcp** [WeblateOrg/weblate#9191] [Ideas]
Hi everyone We a collective of faculty have started a project to develop a Kannada Technical dictionary using Crowd sourcing. Here it is just not enough that we have an English string and corresponding translated string, **For each technical term and its translated term in local language we also need few more columns like English Pronunciation, Description of the term in English and its equivalent description in the local language.**

A sample csv in the proposed structure can be found here. https://gitlab.com/prabodhcp/kannada_tantrika_padakosha/-/blob/main/VTU_FORMAT/PadaKosha_NetworkSecurityCyberLaw.csv

**Is it possible to setup Weblate to have these additional columns for each entry?**

This will be of great help if made possible, as we can extend this project of building a local language Technical dictionary to other technical subjects of our university.
Thanks in advance

**Comment by nijel** [comment]
Answered at https://github.com/WeblateOrg/weblate/issues/9192, please avoid double posting...
